internal/config/config.go

```diff
@@ -0,0 +1,87 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+)
+
+type Config struct {
```

Comment:
Might be worth having a look at https://github.com/kelseyhightower/envconfig for this particular scenario where you want to load all env vars into structs. I've used it in the past and it works quite well. Not sure if it will adapt well to your custom logic (conditionally loading GH app config only if `GITHUB_TOKEN` not specified), but it's definitely worth considering, to reduce boilerplate code.

Reply:
Yeah I meant to refactor this eventually with some known library like that one, I can do that in this PR.